Examples/testCustomEntitlementsComputation/testCustomEntitlementsComputation/ContentView.swift

```diff
@@ -0,0 +1,278 @@
+                Text("Current App User ID:")
+                    .bold()
+                Text(appUserID)
+            }
+
+            Button(action: {
+                showingLogInAlert = true
+            }) {
+                Text("Login")
+                    .font(.system(size: 20))
+                    .foregroundColor(.white)
+                    .frame(maxWidth: .infinity)
+                    .padding()
+                    .background(Color.blue)
+                    .cornerRadius(20)
+                    .padding()
+
+            }
+
+            Button("Purchase first offering") {
+                Task {
+                    await purchaseFirstOffering()
+                }
+            }
+            .font(.system(size: 20))
+            .foregroundColor(.white)
+            .frame(maxWidth: .infinity)
+            .padding()
+            .background(Color.blue)
+            .cornerRadius(20)
+            .padding()
+
+            .task {
+                Purchases.configureInCustomEntitlementsComputationMode(apiKey: Constants.apiKey,
+                                                                       appUserID: appUserID)
+                do {
+                    self.offerings = try await Purchases.shared.offerings()
+                    print("offerings: \(String(describing: offerings))")
+                }
+                catch {
+                    print("FAILED TO GET OFFERINGS: \(error.localizedDescription)")
+
+                }
+            }
+
+            NavigationView {
+                List(customerInfoStreamFires) { customerInfoStreamFire in
+                    NavigationLink(destination: CustomerInfoDetailsView(customerInfo: customerInfoStreamFire.customerInfo)) {
+
+                        VStack (alignment: .leading) {
+                            HStack {
+                                Text("Fired at:")
+                                    .bold()
+                                Text("\(customerInfoStreamFire.date, formatter: dateFormatter)")
+                            }
+                            HStack {
+                                Text("App User ID:")
+                                    .bold()
+                                Text(customerInfoStreamFire.customerInfo.originalAppUserId)
+                            }
+                        }
+                    }                }
+                .navigationBarTitleDisplayMode(.inline)
+                .toolbar {
+                    ToolbarItem(placement: .principal) {
+                        VStack {
+                            Text("CustomerInfo stream values")
+                                .font(.headline)
+                                .foregroundColor(.primary)
+                                .padding()
+                            Text("List fills in below when new values arrive.")
+                                .foregroundColor(.primary)
+                        }
+                    }
+                }.task {
```

Comment:
Nit: I would put this in its own line for consistency with the other methods:
```suggestion
                }
                .task {
```

Reply:
good call